src/sections/TransactionInfoSection.tsx

```diff
@@ -79,6 +95,20 @@ const TransactionInfoSection = () => {
     if (txInfo && !isTxConfirmed(txInfo)) getTxInfo()
   }, 15 * 1000)
 
+  // Compute confirmations
+  let confirmations
+
+  if (txBlock && chainHeights) {
+    const chain = chainHeights.find(
+      (c: PerChainValue) => c.chainFrom === txBlock.chainFrom && c.chainTo === txBlock.chainTo
+    )
+
+    if (chain) {
+      const chainHeight = chain.value
+      confirmations = chainHeight - txBlock.height + 1
+    }
+  }
+
```

Comment:
Let's put this in a utility function at the end of the file, so that we don't need to use comments to explain the code (like the `isTxConfirmed` one) and to improve the readability of the whole file.
```ts
const calculateConfirmations = (txBlock: BlockEntryLite, chainHeights: PerChainValue[]): number => {
  let confirmations = 0

  if (txBlock && chainHeights) {
    const chain = chainHeights.find(
      (c: PerChainValue) => c.chainFrom === txBlock.chainFrom && c.chainTo === txBlock.chainTo
    )

    if (chain) {
      const chainHeight = chain.value
      confirmations = chainHeight - txBlock.height + 1
    }
  }

  return confirmations
}
```

Reply:
Done!